src/Patches/EnemyRandomizer.cs

```diff
@@ -661,6 +641,9 @@ public class EnemyRandomizer {
                     if (SceneLoaderPatches.SceneName == "ziggurat2020_1" && Enemy.GetComponent<Administrator>() != null) {
                         GameObject.FindObjectOfType<ZigguratAdminGate>().admin = NewEnemy.GetComponent<Monster>();
                     }
+                    if (SceneLoaderPatches.SceneName != "Atoll Redux" && (NewEnemy.name.Contains("Crabbo") || NewEnemy.name.Contains("Crabbit"))) {
+                        NewEnemy.transform.GetComponent<UnityEngine.AI.NavMeshAgent>().agentTypeID = 0;
```

Comment:
Add `using UnityEngine.AI;` at the top and
```suggestion
                        NewEnemy.transform.GetComponent<NavMeshAgent>().agentTypeID = 0;
```

Reply:
done